Assets/SteamVR_Unity_Toolkit/Scripts/Abstractions/VRTK_WorldPointer.cs

```diff
@@ -182,7 +182,9 @@ protected virtual void PointerIn()
             OnDestinationMarkerEnter(SetDestinationMarkerEvent(pointerContactDistance, pointerContactTarget, destinationPosition, controllerIndex));
 
             interactableObject = pointerContactTarget.GetComponent<VRTK_InteractableObject>();
-            if (interactableObject && interactableObject.pointerActivatesUseAction && interactableObject.holdButtonToUse)
+            bool cannot_use_because_not_grabbed = (interactableObject.useOnlyIfGrabbed && !interactableObject.IsGrabbed());
```

Comment:
useOnlyIfGrabbed is already used, anyway cannotUseBecauseNotGrabbed is just a local temporary variable used to make the code more readable (in my opinion)